docs/src/assets/scss/no-js.scss

```
@@ -0,0 +1,25 @@
+@media (prefers-color-scheme: dark) {
+    pre[class*="language-"] {
+        color: var(--color-neutral-100);
+    }
+
+    .token.comment,
+    .token.prolog,
+    .token.doctype,
+    .token.cdata {
+        color: #8e9fae;
+    }
```

```suggestion
    .token {
        & .prolog,.doctyoe,.cdata
            color: #8e9fae;
    }
```
Can you do something like this?